### PR TITLE
[algorithms] Replace 'could' and 'might'

### PR DESCRIPTION
**Pull request by jensmaurer**
as directed by ISO/CS.

Partially addresses #4319

### CODE REVIEW

**Review by jwakely — APPROVED** [review]

**Comment by jensmaurer** [comment]
The additional hyphenation rule has no effect elsewhere (checked via diffpdf).

File: source/algorithms.tex [hunk 1]
```diff
@@ -4398,7 +4398,7 @@
 do not overlap.
 \begin{note}
 For the overload with an \tcode{ExecutionPolicy},
-there might be a performance cost
+there is a performance cost possible
```
Comment (zygoloid) [inline source/algorithms.tex:4401]:
```suggestion
a performance cost is possible
```

File: source/algorithms.tex [hunk 2]
```diff
@@ -5296,7 +5296,7 @@
 do not overlap.
 \begin{note}
 For the overloads with an \tcode{ExecutionPolicy},
-there might be a performance cost
+there is a performance cost possible
```
Comment (zygoloid) [inline source/algorithms.tex:5299]:
```suggestion
a performance cost is possible
```

File: source/algorithms.tex [hunk 3]
```diff
@@ -5483,7 +5483,7 @@
     \oldconcept{CopyAssignable} requirements.
     \begin{note}
     For the overloads with an \tcode{ExecutionPolicy},
-    there might be a performance cost
+    there is a performance cost possible
```
Comment (zygoloid) [inline source/algorithms.tex:5486]:
```suggestion
    a performance cost is possible
```

File: source/algorithms.tex [hunk 4]
```diff
@@ -7022,7 +7022,8 @@
 
 \begin{note}
 For the overload with an \tcode{ExecutionPolicy},
-there might be a performance cost if \tcode{first}'s value type
+there is a performance cost possible
```
Comment (zygoloid) [inline source/algorithms.tex:7025]:
```suggestion
a performance cost is possible
```

**Comment by jensmaurer** [comment]
Squashed Richard's fix-up. Needs a fix for the overfull \hbox; working on that.

**Comment by jensmaurer** [comment]
Overfull \hbox is fixed; force-pushed.

**Pull request merged**